Source/Metadata/SvgDocumentMetadata.cs

```diff
@@ -54,9 +47,8 @@ public override void InitialiseFromXML(XmlTextReader reader, SvgDocument documen
         {
```

Comment:
This method is not called from anywhere in this project...

I don't know how it was added from commit logs.
It doesn't seem to work.